host-configs/quartz-toss_3_x86_64_ib-clang@4.0.0.cmake

```diff
@@ -33,7 +33,7 @@ set(MPI_CXX_COMPILER "/usr/tce/packages/mvapich2/mvapich2-2.2-clang-4.0.0/bin/mp
 #---------------------------------------
 # Library Dependencies
 #---------------------------------------
-set(TPL_ROOT "/usr/WS2/smithdev/libs/toss_3_x86_64_ib/2020_04_16_21_23_26/clang-4.0.0" CACHE PATH "")
+set(TPL_ROOT "/usr/WS2/smithdev/libs/toss_3_x86_64_ib/2020_03_25_19_18_28/clang-4.0.0" CACHE PATH "")
```

Comment:
Sorry...

Reply:
The cmake path is right.. but the TPL_ROOT needs to be back to the 4_16

Reply:
Yeah, I'm trying to rebuild the TPLs with mixed success.